sc-tools/sc-builder/src/scs_translator.cpp

```diff
@@ -75,7 +96,10 @@ bool SCsTranslator::TranslateImpl(Params const & params)
   std::string data;
   GetFileContent(params.m_fileName, data);
 
-  SCsHelper scs(m_ctx, std::make_shared<impl::FileProvider>(params.m_fileName));
+  SCsHelper scs(
+      m_ctx,
+      std::make_shared<impl::FileProvider>(
```

Comment:
We need to be more explicit about the file formats we support. For example, move this set to special enum or constants class and describe in docs, while only this formats we can process

Reply:
I think that we can support any binary files. I propose to point all text file extensions, that can we use, and reverse logic in builder  to `if extension file isn't text format than convert to base64`

Reply:
Yeah, it's probably up to the interface/agent/client to decide whether to fetch/show the file based on the extension